server/odc-service/src/main/java/com/oceanbase/odc/config/HookConfiguration.java

```diff
@@ -91,4 +108,20 @@ private void approvalFlowConfigUsageCheck(Long flowConfigId) {
         }
     }
 
+    private void projectReferenceCheck(Long userId, Long organizationId) {
```

Comment:
As long as the current user is added to any of the project, they cannot be deleted. Would this be too difficult to delete a user.

Reply:
I changed the condition, now only if the user is the owner or dba then it cannot be deleted.